### Testing

```
-- The C compiler identification is unknown
-- The CXX compiler identification is IntelLLVM 2025.2.0
-- The Fortran compiler identification is IntelLLVM 2025.2.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - failed
-- Check for working C compiler: /opt/upstream/linux-x86_64_v4/compiler-wrapper-1.0-f75wmeez6pkci4r4cvu7i7k3jhvff5em/libexec/spack/cc
-- Check for working C compiler: /opt/upstream/linux-x86_64_v4/compiler-wrapper-1.0-f75wmeez6pkci4r4cvu7i7k3jhvff5em/libexec/spack/cc - broken
CMake Error at /opt/upstream/linux-x86_64/cmake-3.31.6-cpaa3od6crweusz6ntlfhmvesjodi4un/share/cmake-3.31/Modules/CMakeTestCCompiler.cmake:67 (message):
  The C compiler

    "/opt/upstream/linux-x86_64_v4/compiler-wrapper-1.0-f75wmeez6pkci4r4cvu7i7k3jhvff5em/libexec/spack/cc"

  is not able to compile a simple test program.

  It fails with the following output:

    Change Dir: '/tmp/root/spack-stage/spack-stage-fckit-0.13.2-yg6qrdnjmobuuzaonccpr64iqpax7fkc/spack-build-yg6qrdn/CMakeFiles/CMakeScratch/TryCompile-YbTG94'
```


```
# git -C /opt/package_repos/fncqgg4/repos/spack_repo/builtin diff
diff --git a/repos/spack_repo/builtin/packages/fckit/package.py b/repos/spack_repo/builtin/packages/fckit/package.py
index a88c326c4c..6779aef818 100644
--- a/repos/spack_repo/builtin/packages/fckit/package.py
+++ b/repos/spack_repo/builtin/packages/fckit/package.py
@@ -30,6 +30,7 @@ class Fckit(CMakePackage):
     version("0.10.0", sha256="f16829f63a01cdef5e158ed2a51f6d4200b3fe6dce8f251af158141a1afe482b")
     version("0.9.5", sha256="183cd78e66d3283d9e6e8e9888d3145f453690a4509fb701b28d1ac6757db5de")
 
+    depends_on("c", type="build")  # generated
     depends_on("cxx", type="build")  # generated
     depends_on("fortran", type="build")  # generated
```
 

```
==> Installing fckit-0.13.2-ahec6hb2drx3o4dtriipy2lkvvzrcf5r [45/46]
==> Using cached archive: /opt/sourcecache/_source-cache/archive/99/990623eb4eb999145f2d852da9fbd71a69e2e0be601c655c274e8382750dfda2.tar.gz
==> No patches needed for fckit
==> fckit: Executing phase: 'cmake'
==> fckit: Executing phase: 'build'
==> fckit: Executing phase: 'install'
==> fckit: Successfully installed fckit-0.13.2-ahec6hb2drx3o4dtriipy2lkvvzrcf5r
  Stage: 0.03s.  Cmake: 3.41s.  Build: 13.69s.  Install: 0.12s.  Post-install: 0.03s.  Total: 17.35s
[+] /opt/upstream/linux-x86_64/fckit-0.13.2-ahec6hb2drx3o4dtriipy2lkvvzrcf5r
==> No binary for fiat-1.2.0-z5gldmzzuz7eoahwndwecrxzy64fseah found: installing from source
```

--- a/repos/spack_repo/builtin/packages/fckit/package.py
+++ b/repos/spack_repo/builtin/packages/fckit/package.py
@@ -30,6 +30,7 @@ class Fckit(CMakePackage):
     version("0.10.0", sha256="f16829f63a01cdef5e158ed2a51f6d4200b3fe6dce8f251af158141a1afe482b")
     version("0.9.5", sha256="183cd78e66d3283d9e6e8e9888d3145f453690a4509fb701b28d1ac6757db5de")
 
+    depends_on("c", type="build")
     depends_on("cxx", type="build")  # generated
     depends_on("fortran", type="build")  # generated
 

